### ISSUE TITLE
energy blade

### ISSUE THREAD

**Issue by roroah** [PathOfBuildingCommunity/PathOfBuilding#3731]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

When i activate the energy blade gem there be se somthing about my weapon being an energy blade

### What is the actual behaviour/value?

Nothing happen when you set the gem

### How to reproduce the issue

Set the energy blade gem in you loadout gem

### Build code

```shell
eNrtXelz4kiy_zz-KxSO2Bc7MWBLpdvPvRuA7zY2Ddju7i-OQipAtpCwDg5PzP--WVUCxOmy2t2xb-LNRPcgkZmVV_0yq1Rijv89GfjSiESxFwaf9pUDeV8igRO6XtD7tH_XPitb-__-195xAyf922419Xz6zb_2fjtmnyWfjIj_ad_W96UERz2S3M9EqY8gaoiDpE_CoI6fwug8dD_t34QB2Zc6OHC9ZHbl-DiOb_CAfNpvk8HQx9G-hGOHBG5t8c1l8JJ6sZeE8OUAe0ErdJ5Jch6F6RD03pdGHhnXQ5dS1hu3zXZucC_IDw7K_3bc8PGURK0EJ1IMf33ar4APcI-c4AH8DdKwn4IoxZJtdKDISLZtw0L7h5uZW0NC3DkTOlAN_ZfTXnjJMrl6oCB9G3UjIqfdLnESb0RqkZfU-jhwFnYj60BTt9u7iUM_UMFJtrWLp576iTf0PRItPHxgmztMWhnFsrfRtsME-yeN1pxUN3TVPLAUWzYUXbZ-Dl-YzPnkbZQPXtKv-hCZAqNQ3ste4CWkIHMj9OIw-AH78qxbTayFg44XFLJwxloZ9d7LWokIvu3yNG5i10vjRTZuVbWOA1wL40XcFGUb6bXXJUukW4WetsTomoAtYpRUzQaJAAMTMQaq7LsYWkluEurqVpXJyzLlVkg5IZMFGVJ2CFyiVLdqeBkszFCNXRrmKXdqOAoTKE5i3pxTaQrSdiC0c0SpLwMnZ5O6S_BdEJGYRKMcVu8aYpkli_FiLHmnEU3SIwt7Te0AqWgX_TUhTv8cymsTJ0QsTXOq2MpOP1HqvJ9Ue5fcDW5SkHxgq2JMa46C3sbSVGSogCqaskvIstNU42DnkO902QOOXAFECUjUm7b6HvFzFV2zzJ0OznMtJaSliAy0bLeQbu80_nSE4_wEVOzd9nDyvCnbU5FAjQUGl6z0DB_I0YjCJ9oz-e9jq0SDMI1yk91EO83m9EsB3O6oRn8aew7Uata5NombOksYp20PxqwBrIcjMoC5wvpH6KFzzdn2DrLqQwMu6gGQ7Pvv4qgkCXaeT0K3R941yPs5WulwCKBBE02U78yLwNOxl6uzpv428S2sMWp4uODZ3hTRybgsH71NvCp_R2Xo9ZMAFnIrgxiWIIfwSODRMF4Zpbx9EZMjFx7iLPV9oS619ewtUya445MjSZ4oDnGwub2SNsIxKNOnC9xYACVy1HW8aHW2VpGziASvU2H5S-RCA5wGbhrR1BYeY5VjfZjjQ7YDQD9dDoZhlLCbNew7MRN5GQzTRArY2n3gxc5jJ-126QJ9H4aI2PbC6dnZaa19eX-aaZFniWmwHoN00KGLRf5fus7nlC3CQE5yQt_Hw5jA2r6L_Rhke24W6Bagu5MI0cM6M1v3i1BDAXciD5woQkxXiEIqs6W7CCVdRwsRQlSwT8Qd1p4OCc2AWNRnGVqKULP1rxAlX7CKGcgWxmJhgPk9FaKcNzRC1Kc-qXg-rZ1iXqvDROBlV4weqmDkddJEMNvYskVID9q3CxmY70MFZ4cYYdYUCSmRdY0itFmtEUtK6EeEw3ZCugTSXXDaMzhp42cSZGA5A8ZjNtdiKQbEPCeDuDqFinRG3buyjbJEALUqSkG2S7o49en9Lyn2vWRKITx395pvxyK4GffDMW1tuBg6ueNP-ze3zXrlmn9Z8ZNMCB1mpj43jurIdlkrrEFk18wCttXqBY6furDkzUrv3HQfd-jw-1KPbsvWwjSAUhJ4Pt1TprXWndmxIppKpQP_dgyqZLTnftjBPlrIXpiWJ1BmIlnBuKRgBhjmOS2YOM9w-4WbeMlAgLkpI23gCJRDoCsZ0G_rJMEuTvDhZQLuOKQ-OWTKwaeNIpnrV-yaS144y-FegE-0sNH-fqaiNBPIuOpewPbNW-lgEEISNABdExwPeL59jGd4Osx7uQYJSBJhvlzY5Khdrpmn1k5xy07KFN7kkrkUKS_mw21_IHgYBgDbFLNni6YfMH63vE0pssn4Ob_EBUhjL-lLfAkUf7wTzuCP153-gNmrEkQNnfF9uEnNNOn7sIj7AZvWRIgaNWf8-EDhOCERz4M3oUwRjd0mocIBZMzz1NwMXU2YFCT6CUGmmxvE5ZuOxeO8QYqo-ZdBPPTy2HTMa8T__ap5G3Vuu60kjAbLUaUubBHuhw-ppUsD_XgpBXFS2JU2aP73LqXbc_QXVVKOBDVMN3aK15F1Ie9EIsoLtn68ffOa_DGh3Snu_X3CTw1txXWJO0-jH-6RdooTNZ0JkRa5nd-w-f8a899QYxTR7IocHJBWGvXIh_U0m2QKZxbjlTLmX9PQnPkw_AmO-z_qgayorssThlHKKXHWX1U8a2HAnlKNQM_3T7LM5I1CRK1eYv44w76knvN8nrInvAXNaiWE-CDw_cVhwfn36kwrjpNG2JlWApfufccVuCrs3kZEHC8uUnsXnB-XLw8A8x8FAcuyRK3KuH4N6l2QCPvubbfdTwOXPtb5EMO3SRV1Aeeni4i5hKJAyAfZ7o_MYXOH3GPsn3ixQ89EBuSD_LFFqKg7KLuU5192xrOXQNvdCRP62PlvhjQNoZ5EbJnbKNSLPOARoZkI9WnkZec4flVVruIkgfUFdNVxLSpel7eIEfXAgv1_8DCM_zeWqJQPb_Nvg9jHKSwmboOWj6ewomj16YFy9zQggx_ZGRUULL4YCEg0wslPWOlAjtEDclAAQDn2TPIHzN4h7B092VyElMn49fgS-yE9U0qXmMrPRRv-aLXqY5cUnmybZAhvJjBeiTOvODr7xB5YtiNCJMydwBiV7CEhXORf06C6MpfRfZlp9pT70_6fqmYqJWTrlvFX6U_V1FVU0mXbVuEKvrHNkqqpFr1CmlwyNNmgn3XLUoySrsoa5TJUzULwHdKUv8CE0KXPMk3VskqmrVolBSFdLqmWKSslU9HVkoZMQy7ppqaWkIyQVeICkKmoMJwBVyUN_pJLmqZYKtwxS7phwoBAZQO1oiL4TtFMVFJ03bZLSLdUMMLQdaDUVTBBsSxDB0oDhjMUHelAj3SlpNuqXlJk3dTBZlmG26pi0yEUpJQ024A7hmqCSBXZ1BNghFZSNBm-RQo4qaTaBgwFtipyCW6ASYZMFYE_IJ77TwWf2jAs9axqahZTXNGoBMTuKzYd3QCZYKSqmzL40lQNUFtVDK1EbQK1kQHeR6am6OA8SzXAKNOgd2Qd7uhUGwW0ARtYnAyTsiIbRtNU1aZagzsMBD5V4C-4AbqDB3joQCELBkIaeJwe5Qd2Bey1dB34DYvKg-GUkmICJXjDBHuRAWODGIs6EgJHDYXhDaq4DF7RNOY5-AZ0kiGQ1FuaVYIkAOWRpdMBTAOiroCokgYyQIQJ1kD2yRpEwdbskinb1J0qqAMOM6n7DYilijQgVCwYVrE1xWTesEEPGXyqqirox1JZYYFAiqpCioHeNJQayw7LoK6BjAPldBnyCBIDlFCpLFPVaTIp9LOsgTt0nZIjMMYsgYY0UWga6gY4zJBVSEMVggETKoH5l3urSjGyF6bofNMzODp1PUDNGzoxsjpx17xmH37rJ8kwPjo8HI_HB0No9sMumXg-OXDCweEQxMC0LjNMKtOBDivwT7V3V-32X52L8eUIDZ3k_PE8qFW_j9rjtloehNWnyYtS_Tz9Wnl4dJ-upy-m5nmVk9fR91s3GLm951bnYqzg06ezqTVsq2c3L9eDm3Pn4qzcJpXhzZV1E9_fT4ZRo5xc1a_JWEuvblvB5fWFVS23OtHk4R4HI0M9P6s0atckrbTq6BWn3fYQT8jX5BTaMFyuu8_T1_RMxp1qaxy168ho9t36WYw88rWioejrq36Xnj89PY-unUnzu_I4GWBy04_bQfi5Uxl-O60a5qPbP3ejYdS7-hxOb24fru_GWtMPGx39dvrlvpnK6U31Xnnyr7qxc3HxPPreur846Xy7JCff7JNvyk1LrtrlCD1No88vpFENvsZf7k-mJ_HN50aEb_3g9eJbvXli-12jbLSvxpXz8uhCLcfO5Hn4ajTGJ-P-c1n76lhId6qvzz3UYIE7nEXumL_hFs-gGeB2gbuJB13Sp_0hPWq4L4LCfFbCFDHsFahdBuHtUJtHWQ5bmUyGsAx3OcAynOCAsoqvhmqbRgY_HF8ZfPBZsAQWMH05uPKZtwauHAo4CnOs4eCaTXQOrhRC8tjKkZeDOcdWPsc5tnLYz2MrAwMOv-vQyiGXA3seXHllycAVLFIzZNgNrhyG19CVl588vHI842CVL0gM7HixEYFXhsYcXnmd4gDLwCoPsKyIqhqAF4faPL4y2GXYxYsWh1eOejl45bWMJxavzBxfeeXi-EoTktfyLejKi1geXXkp4cDK6_Y6vPLKzCvDfxm-Vl-da_vl67V7ZcH8b1SrownuneLOFZGjK6fasVG9TJ5wii8r52fKd8vuNvyWVvNHJ_e9vtG_nDijz3fl9FEeTADvqhPSjnufv7SDp87ZvdPRb1A7aNeUq7uO1wZ8rk0eK93qxIxvzx6GZjmZ9DuTh6dm0Dt9uomTL23ZuO9e4cbVVRsNRk8X3-ux2XfxxLmLP8uvN-mzZb5WOzfh4AQy8eK7qX67qp4_AK6r9bsHt2t9HwRP3zrnT15tAHhfrZBJ9eu3ZoC1h85YsQbpie8p9VvAWbenR43ns7vvivn05d7tGPKoawzH_Wv0_SodI7liVa3aF3TTvksq3e5L-Zq8TF-Cp9fn0WnjcvQ5NE9aTwZ6AryunjbQ_WNkDutBpdIcDQD3GZ52o4n9GDF8FcHT40Pa39IPN2FC2NFlenN2wbrfe4-MpZjgyOm3kujT_pieAIa13GsYDr4B3GoIHRimrdsAFwjx03UXBCd1PJwvCyhtdjrPzg7nJTg58QCgI3aucNaqU8KvdP1gHkCXDAgEaGLyl8OO2Qog68XzD-bTmLSIEwYuP47EbueO81FSdiwR8XRu4gjWCUdSs9I83ePnMA-z1QA_6SlVwzCJ99rjsNymuw38WvrnJtrfMwlHEtK07HMVx7OXET12wn5vieVI0md3-I11-lqEuzDjjiTqlL1GRLre5Ei6Dh3sXwZORIDB5WNVAjcvSp8Tz-noqVhjfnv2sss98UMH3KDvtdIu-4a-Hb7jIjssCXbKe1kqHUnVMvt3j4W2SV6OJFPeuxwMfQ9kw_fK3p8RDnrkSD7Q__rjn1ZZQb__Q0pCib4VIuEg_0iTH12lZ6rjPVP7B13McgMkbikjX3Lk3h-GToUN8MQbpAOJWrqnynnWmbkSO2HO5kE9dJtUJ4lpBsvFA52fWp2drmcrzeXMUUUzp9YncbLXgnVz4k_hCj7AItqfJ4kiy6polkCNKZ4mlPiNDCmeTvZHJM3OxJH3_lAUgwaX6yWcEaqdz4iVL-31dNkcb0003hfEH5AEvIRJEkZSLQrHwTzY0B-IxlpRtZ8Qa_UDYm3OI5pNZeWvhZzs2dQNGaurZHTGnxB_RLiL2P7fymtUdK7Tx1lKIaQxrJWEgf6cJUymlNTE9GgMpIRVII8Ua0cemet5dAP1sTOlhAOPxFKf7mSXbYZ1myBuS97ponl37gOwxZAUaYxdEs2u57XIRKKZZxg_IfHQ44cUrbXMOyETWNctla33ZyU0MSToJX1dONOMlUxTWaLN5MA1otdz5banomIXSMVdiGaIIpqxKbMGqQ_gdQUZJGWfNwc757ylCGk7KfIhzlJqIwNDiSsyJn40XTkntUCG7ZGnIlbYstHU9SRZvC2q5QKs6juaFiSXVfn3lQDzDFjc0Ja6jtVzWnsKyn-9JZbaeiz_QBxCqNrL6FGsjzE3ZMFCrET3_SFCXjxgL2iwyzczYuM7AKpwZhirlMpyji2CuHiTWJvfy8dzdm-lyiwF-o3uVObdKfZ9aXHYMN-Y_oGUHbNxufnIImguGt5cBGf3VyM7u7-xaCAjn0a7XpuQ-AOMwolibUiUlTUS8QEuaIObfdyYJ4VCXhwlttBv1OIsCgfV0J1y4N0ug86ClexW1oEln5yLfEPWjnyzy4jjyuZsQvpOSNmdiZq9FUtWMxFZu5JqRY3lHOiCCyXqw1lJK5puyqZ0m2uCLqTWOIzcvSoOvLhPncAe223JuY2AdBvAcr6PZ3sEixRjLefSxRxHljsXNq_YMlJdJqHHsuhPUPD32fhrwVtIFr8Apm1uPs7L8393LKqzOT6Tmr1JF0teLwghuKzXhqVvwI6Ibway5Yo0Aw22SEZLX60MInH75jjFH--uUS0M3TPM94NWLkdafjh7KZ6Hjj6Y9hL-cHj-QvyCiHfCCxJ9nYTi1YLAWifgS5YFibZOwh6RowWJuWGYxcRY0KlbRClviAqTnE1onYL3bwsSY_ajAnyfbMceWX4CbvJ2a4yHUqUzjWMaXpaokr4YSJ4dzdjAusqGxNjW4rvui4IKbbLlDaVosryT5czH8bOkCqmENqmkFnOvmBPQB7FpxZQ0io1mFPem8kaAN-XTGxHgAPHegTZkklFknDfSbwsCCghWi88ipYghWrFcUIplnvZ2eNaqwgbHvlfspvQqaLgQ7ugF9CmAgepbw8yL3zvDt9nLhfJLL8IkNCXXeoMNgShkp17YrahA3I1iM0ktlr7q-7NMKw77BXJaKex8VWRmaiJEBdT-geZHMJLFo2AUyMriYfiBdBGFZMorOGvESQt4bssCRABiUGH3FoQL8XiurZMEzFEF_KTs9tOuMGiFvaUXwX-R-bK2EBRO8jdU2raIPMxWkezACDuFwX7sLgy6Xm_tp-voItOj727QPYEmoRvx_nRf6oShT3CQnfNY_8U7QjctLuNqSH-CZPYTea3-4h2s5Z_Um-8v0Dfgr-GPxZe38eLX8tQNfN7shYLZmwXsHZO3dBvjEbntLt5KOZ0MwziNCP1ZqYWy8y26DRLmTsm_DiHuHPrjgWc4oLvyXjx4m34-HBsnG_MdbOy3wegrcNsVPD6cxf_4cPV_j_Af8b3t7A==
```

**Comment by aeroncastle** [comment]
have the same problem, is there any workaround while POB is not updated?

**Comment by Wires77** [comment]
Fixed in 2.14.0
